doc/news/changes/major/20190513Many

```diff
@@ -0,0 +1,5 @@
+Improved: All tutorial programs have been reviewed and updated with new features
+in the library and C++11.
+<br>
+(2019/05/13, Daniel Arndt, David Wells, Matthias Maier, Denis Davydov,
```

Comment:
isn't the format "author, author, date"?

Reply:
@tjhei fixed it for David